lib/task/digitalobject/digitalObjectLoadTask.class.php

```diff
@@ -267,6 +271,12 @@ protected function attachDigitalObject($item, $informationObjectId)
     $informationObject = new QubitInformationObject;
     $informationObject->parent = QubitInformationObject::getById($informationObjectId);
     $informationObject->title = basename($item);
+
+    if (property_exists(get_class($informationObject), 'disableNestedSetUpdating'))
```

Comment:
Why do we need to check the property existence?

Reply:
We don't I suppose now that I look at it again.  I can remove this.